2021-01-19 21:24:27,792 INFO    MainThread:27499 [wandb_setup.py:_flush():70] setting env: {}
2021-01-19 21:24:27,793 INFO    MainThread:27499 [wandb_setup.py:_flush():70] multiprocessing start_methods=fork,spawn,forkserver
2021-01-19 21:24:27,793 INFO    MainThread:27499 [wandb_setup.py:_flush():70] setting login settings: {}
2021-01-19 21:24:27,793 INFO    MainThread:27499 [wandb_init.py:_log_setup():303] Logging user logs to /home/fkubota/Git/sand-pit/028_wandb_kfold/wandb/run-20210119_212427-aot1km3y/logs/debug.log
2021-01-19 21:24:27,793 INFO    MainThread:27499 [wandb_init.py:_log_setup():304] Logging internal logs to /home/fkubota/Git/sand-pit/028_wandb_kfold/wandb/run-20210119_212427-aot1km3y/logs/debug-internal.log
2021-01-19 21:24:29,247 INFO    MainThread:27499 [wandb_run.py:_console_start():1237] atexit reg
2021-01-19 21:24:29,247 INFO    MainThread:27499 [wandb_run.py:_redirect():1104] redirect: SettingsConsole.REDIRECT
2021-01-19 21:24:29,248 INFO    MainThread:27499 [wandb_run.py:_redirect():1107] Redirecting console.
2021-01-19 21:24:29,248 INFO    MainThread:27499 [redirect.py:install():204] install start
2021-01-19 21:24:29,249 INFO    MainThread:27499 [redirect.py:install():219] install stop
2021-01-19 21:24:29,249 INFO    MainThread:27499 [redirect.py:install():204] install start
2021-01-19 21:24:29,249 INFO    MainThread:27499 [redirect.py:install():219] install stop
2021-01-19 21:24:29,249 INFO    MainThread:27499 [wandb_run.py:_redirect():1151] Redirects installed.
2021-01-19 21:24:29,250 INFO    MainThread:27499 [wandb_config.py:__setitem__():135] config set dropout = 0.2 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f12313aa400>>
2021-01-19 21:24:29,250 INFO    MainThread:27499 [wandb_run.py:_config_callback():549] config_cb dropout 0.2 {'_wandb': {'cli_version': '0.10.12', 'python_version': '3.6.9', 'is_jupyter_run': False, 'is_kaggle_kernel': False}, 'dropout': 0.2}
2021-01-19 21:24:29,289 INFO    MainThread:27499 [wandb_run.py:_atexit_cleanup():1207] got exitcode: 0
2021-01-19 21:24:29,290 INFO    MainThread:27499 [wandb_run.py:_restore():1179] restore
2021-01-19 21:24:29,290 INFO    MainThread:27499 [redirect.py:uninstall():223] uninstall start
2021-01-19 21:24:29,291 INFO    MainThread:27499 [redirect.py:_stop():278] _stop: stdout
2021-01-19 21:24:29,291 INFO    stdout    :27499 [redirect.py:_pipe_relay():120] relay done saw last write: stdout
2021-01-19 21:24:29,292 INFO    stdout    :27499 [redirect.py:_pipe_relay():136] relay done done: stdout
2021-01-19 21:24:29,292 INFO    MainThread:27499 [redirect.py:_stop():284] _stop closed: stdout
2021-01-19 21:24:29,292 INFO    MainThread:27499 [redirect.py:_stop():290] _stop joined: stdout
2021-01-19 21:24:29,292 INFO    MainThread:27499 [redirect.py:_stop():292] _stop rd closed: stdout
2021-01-19 21:24:29,292 INFO    MainThread:27499 [redirect.py:uninstall():227] uninstall done
2021-01-19 21:24:29,292 INFO    MainThread:27499 [redirect.py:uninstall():223] uninstall start
2021-01-19 21:24:29,293 INFO    MainThread:27499 [redirect.py:_stop():278] _stop: stderr
2021-01-19 21:24:29,293 INFO    MainThread:27499 [redirect.py:_stop():284] _stop closed: stderr
2021-01-19 21:24:29,293 INFO    stderr    :27499 [redirect.py:_pipe_relay():120] relay done saw last write: stderr
2021-01-19 21:24:29,293 INFO    stderr    :27499 [redirect.py:_pipe_relay():136] relay done done: stderr
2021-01-19 21:24:29,294 INFO    MainThread:27499 [redirect.py:_stop():290] _stop joined: stderr
2021-01-19 21:24:29,294 INFO    MainThread:27499 [redirect.py:_stop():292] _stop rd closed: stderr
2021-01-19 21:24:29,294 INFO    MainThread:27499 [redirect.py:uninstall():227] uninstall done
2021-01-19 21:24:30,032 INFO    MainThread:27499 [wandb_run.py:_wait_for_finish():1324] got exit ret: uuid: "33ccc5989edf45e680e6d220b6fe62f5"
response {
  poll_exit_response {
    file_counts {
      wandb_count: 1
    }
    pusher_stats {
      uploaded_bytes: 695
      total_bytes: 695
    }
  }
}

2021-01-19 21:24:32,037 INFO    MainThread:27499 [wandb_run.py:_wait_for_finish():1324] got exit ret: uuid: "5c93ad75770642cca3c8e3f327d8aae4"
response {
  poll_exit_response {
    done: true
    exit_result {
    }
    file_counts {
      wandb_count: 4
    }
    pusher_stats {
      uploaded_bytes: 4372
      total_bytes: 4372
    }
  }
}

2021-01-19 21:24:33,276 INFO    MainThread:27499 [wandb_run.py:_show_summary():1469] rendering summary
2021-01-19 21:24:33,279 INFO    MainThread:27499 [wandb_run.py:_show_history():1507] rendering history
2021-01-19 21:24:33,287 INFO    MainThread:27499 [wandb_run.py:_show_files():1536] logging synced files
2021-01-19 21:24:33,291 INFO    MainThread:27499 [internal.py:handle_exit():138] Internal process exited
